### ISSUE TITLE
Ignore parts of code in a final CSS file

### ISSUE THREAD

**Issue by emagnier** [CSSLint/csslint#406]
I'm using Sass to compile my CSS, and also use a CSS framework (like [Bootstrap](https://github.com/twbs/bootstrap), [Foundation](https://github.com/zurb/foundation) or [Inuit.css](https://github.com/csswizardry/inuit.css/)).

When I'm using CssLint with Grunt on my final CSS, it gives me some errors / recommandations on the CSS code from the Framework.

Because I don't want to modify these frameworks / libraries, it should be good to add special comments to define certain regions CssLint will ignore.

e.g.:
````css
/*# cssLintIgnoreArea=begin */
@import "../bower_components/bourbon/_bourbon.scss";
@import "../bower_components/inuit.css/_inuit.scss";
/*# cssLintIgnoreArea=end */

@import "_my-custom-code.scss";
````

With this I will be able to use efficiently CssLint on all my Sass projects.

**Comment by marcochavezf** [comment]
+1

**Comment by bdgamble** [comment]
+1

**Comment by lorenz068** [comment]
+1: I need this feature too, please :

**Comment by Newnab** [comment]
+1

**Comment by himedlooff** [comment]
:+1:

**Comment by frvge** [comment]
I think PR #564  will do what you need.

**Comment by laurentperroteau** [comment]
+1
